pipeline/alert.go

```diff
@@ -936,6 +941,91 @@ type SlackHandler struct {
 	Channel string
 }
 
+// Send the alert to Telegram.
+// To allow Kapacitor to post to Telegram,
+//
+// Example:
+//    [telegram]
+//      enabled = true
+//      token = "123456789:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+//      chat-id = "xxxxxxxxx"
+//      parse-mode = "Markdown"
+//	disable-web-page-preview = true
+//	disable-notification = false
+//
+// In order to not post a message every alert interval
+// use AlertNode.StateChangesOnly so that only events
+// where the alert changed state are posted to the chat-id.
+//
+// Example:
+//    stream
+//         |alert()
+//             .telegram()
+//
+// Send alerts to Telegram chat-id in the configuration file.
+//
+// Example:
+//    stream
+//         |alert()
+//             .telegram()
+//             .chatId('xxxxxxx')
+//
+// Send alerts to Telegram user/group 'xxxxxx'
+//
+// If the 'telegram' section in the configuration has the option: global = true
+// then all alerts are sent to Telegram without the need to explicitly state it
+// in the TICKscript.
+//
+// Example:
+//    [telegram]
+//      enabled = true
+//      token = "123456789:xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+//      chat-id = "xxxxxxxxx"
+//      global = true
+//      state-changes-only = true
+//
+// Example:
+//    stream
+//         |alert()
+//
+// Send alert to Telegram using default chat-id 'xxxxxxxx'.
+// tick:property
+func (a *AlertNode) Telegram() *TelegramHandler {
+	telegram := &TelegramHandler{
+		AlertNode: a,
+	}
+	a.TelegramHandlers = append(a.TelegramHandlers, telegram)
+	return telegram
+}
+
+// tick:embedded:AlertNode.Telegram
+type TelegramHandler struct {
+	*AlertNode
+
+	// Telegram user/group ID to post messages to.
+	// If empty uses the chati-d from the configuration.
+	ChatId string
+	// Parse node, defaults to Mardown
+	// If empty uses the parse-mode from the configuration.
+	ParseMode string
+	// Web Page preview
+	// If empty uses the disable-web-page-preview from the configuration.
+	IsDisableWebPagePreview bool `tick:"DisableWebPagePreview"`
```

Comment:
Can you add a `// tick:ignore` comment line to the `IsDisabled*` fields so that the auto gen docs skips these place holder fields?

For example

```go
// Web Page preview
// If empty uses the disable-web-page-preview from the configuration.
// tick:ignore
IsDisableWebPagePreview bool `tick:"DisableWebPagePreview"`
```